src/sentry/services/hybrid_cloud/import_export/service.py

```diff
@@ -60,6 +60,7 @@ def import_by_model(
         filter_by: list[RpcFilter],
         pk_map: RpcPrimaryKeyMap,
         json_data: str = "",
+        min_ordinal: int,
```

Comment:
Adding required parameters isn't a safe operation. If there is any active customer traffic on this you'll need to do multiple deploys to get to a required argument.

1. Deploy the new signature with an optional parameter.
2. Deploy callers that define the parameter.
3. Make the parameter required.

Reply:
Given that we've had no admin-assisted customers so far, I am okay with going ahead if you would be? Would like to avoid the long deploy time of multiple PRs if possible.

Reply:
> Given that we've had no admin-assisted customers so far, I am okay with going ahead if you would be?

If there is no customer or internal usage of relocations yet, lets go with what you have.